website/docs/r/compute_network.html.markdown

```diff
@@ -86,7 +86,7 @@ The following arguments are supported:
   it will create a subnet for each region automatically across the
   `10.128.0.0/9` address range.
   When set to `false`, the network is created in "custom subnet mode" so
-  the user can explicitly connect subnetwork resources.
+  the user can explicitly connect subnetwork resources. Defaults to `true`.
```

Comment:
This appears to be [correct](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/products/compute/terraform.yaml#L1407), however this description is auto-generated and needs to be changed upstream. Specifically here: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/products/compute/api.yaml#L7924.

Reply:
Thank you for info. I moved PR with default value for `auto_create_subnetworks` to `magic-modules` repo.